app/controllers/api/v2/containers_controller.rb

```diff
@@ -7,7 +7,7 @@ class ContainersController < ::Api::V2::BaseController
       resource_description do
         resource_id 'containers'
         api_version 'v2'
-        api_base_url '/api/v2'
+        api_base_url '/docker/api/v2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.